### ISSUE TITLE
Finish documenting MineTweaker wiki

### ISSUE THREAD

**Issue by Cronosus** [MineTweaker/minetweaker.github.io#2]
i was searching for mt wiki, and was for a moment happy theres backup.. unfortunately this is basically just empty wiki with only first page. 
theres missing lot of pages
if youre doing backup, please finish basic pages

**Comment by Lylythii** [comment]
Hi! I do plan to finish archiving it. Just ran out of time & got distracted with other things.

For the meantime, do check out the linked Archive.org page!

**Comment by Cronosus** [comment]
yeah, i did that, as i was in grave need to check some formats for recipes. :D
